test/nightly/azureResourceWebAppActions.test.ts

```diff
@@ -114,6 +114,14 @@ suite('Web App actions', async function (this: Mocha.Suite): Promise<void> {
         assert.equal(await getAppSettingValue(resourceName, resourceName, appSettingKey), appSettingValue);
     });
 
+    test(`Delete setting for ${WebsiteOS0} Web App`, async () => {
```

Comment:
I think I'd prefer if this was combined with "Add new setting for ${WebsiteOS0} Web App". They're both pretty simple tests and this one won't work if that one fails.